ui/plugins/cloudian/cloudian.css

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
@rhtyd, As there is no content in this file, Can we remove this or do you use it for any other purpose?

Reply:
For sake of completion, all plugins have this file and was put for the `cloudian` UI plugin. `require.js` tries to `GET` this file, when this is missing it causes a HTTP 404, however, we'll need to test if it causes any rendering/load issue in the UI. You may remove this after performing this test.